src/libraries/System.Private.CoreLib/src/System/DivisionRounding.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    public enum DivisionRounding
+    {
+        Truncate = 0,
```

Comment:
Nit: assigning underlying numeric value isn't required for implementation source.

Reply:
Similar enums include it: https://github.com/dotnet/runtime/blob/da8a603593d7e0622871554b6bacdf69d22a11cc/src/libraries/System.Private.CoreLib/src/System/MidpointRounding.cs#L6